doc/api/toolkits/mplot3d.rst

```diff
@@ -4,8 +4,11 @@
 mplot3d API
 ***********
 
-.. contents::
-   :backlinks: none
+.. note::
+   `.pyplot` cannot be used to add content to 3D plots, because it's function
+   sigantures are strictly 2D and cannot handle the additional information
```

Comment:
```suggestion
   `.pyplot` cannot be used to add content to 3D plots, because its function
   signatures are strictly 2D and cannot handle the additional information
```